pages/EC-WebSphere_help.xml

```diff
@@ -2733,6 +2733,10 @@
                 <img src="../../plugins/EC-WebSphere/images/ModifyApplicationClassLoader/EC-WebSphereModifyApplicationClassLoader4.png" />
             </div>
             <h1 id="rns">Release Notes</h1>
+            <h2>@PLUGIN_KEY@ 2.3.1</h2>
+            <ul>
+                <li>Internal improvements for 8.3 release.</li>
```

Comment:
This release note is too vague. Instead, lets add these depending on the change being done. The actual notes to add to the release notes section are the parts in quotes.
For sub-stories of CEV-17708 -> "Configured the plugin to allow the ElectricFlow UI to render the plugin procedure parameters entirely using the configured form XMLs."
For sub-stories of CEV-17558 ->  "Enabled the plugin for managing the plugin configurations in-line when defining an application process step or a pipeline stage task."